source/opt.d

```diff
@@ -18,18 +19,25 @@ struct Options {
 		}
 }
 
+class NoInputParams : Exception
```

Comment:
No já bych to možná pojmenoval ArgumentsError nebo tak něják a pak bych v případě potřeby od ní dědil (běžná praxe).